ctms/acoustic_service.py

```diff
@@ -299,9 +299,9 @@ def transform_field_for_acoustic(data):
             return "0"
         if isinstance(data, datetime.datetime):
             # Acoustic doesn't have timestamps, so make timestamps into dates.
-            return data.date().isoformat()
+            data = data.date()
         if isinstance(data, datetime.date):
-            return data.isoformat()
+            return data.strftime("%m/%d/%Y")
```

Comment:
https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes



%m | Month as a zero-padded decimal number. | 01, 02, …, 12 | (9)
-- | -- | -- | --




%d | Day of the month as a zero-padded decimal number. | 01, 02, …, 31 | (9)
-- | -- | -- | --




%Y | Year with century as a decimal number. | 0001, 0002, …, 2013, 2014, …, 9998, 9999 | (2)
-- | -- | -- | --